src/pages/FlowRun/Actions.vue

```diff
@@ -183,7 +183,9 @@ export default {
             depressed
             style="height: 46px;"
             small
-            :disabled="!hasPermission('update', 'run') || !canRestart"
+            :disabled="
+              !hasPermission('update', 'run') || !canRestart || restartDialog
```

Comment:
is this change necessary? that dialog is a modal, so the user shouldn't be able to click anything behind the modal when `restartDialog` is true

Reply:
although, visually I do like the effect